modules/ROOT/pages/6.0-release-notes-premium-changes.adoc

```diff
@@ -108,6 +211,69 @@ NOTE: There is an open https://github.com/tinymce/tinymce/discussions/7487[reque
 If a community-maintained version of the `paste` plugin becomes available, we will link to it from the xref:migration-from-5x.adoc#plugins-paste[_Migration Guide_].
 
 
+=== Real-time Collaboration
+
+The following improvements, changes, and fixes were made to Real-time Collaboration.
+
+Three additional options are now supported when Real-time Collaboration is active:
+
+* `valid_elements`
+* `valid_children`
+* `extended_valid_elements`
+
+Real-time Collaboration now supports the `autocompleter()` API.
+
+//xref:autocompleter.adoc[`autocompleter()`] ?
+
+Real-time Collaboration now supports the `inline_boundaries` option.
+
+// xref:partial$configuration/inline-boundaries.adoc[`inline_boundaries`] ?
+
+When provided values are incorrect, the returned error messages are now less opaque.
+
+For example, the error message `'key' was not a string` is now the error message `Expected field 'key'`.
+
+Previously, if a {productname} editor instance with Real-time Collaboration loaded also contained a Horizontal Rule (ie an `+<hr>+` element), attempting to move the insertion point over the Horizontal Rule using arrow keys threw an exception: `Uncaught Error: Unsupported node type: [object DocumentType]`.
+
+With this update, moving the insertion point through a Horizontal Rule using arrow keys works as expected: the insertion point moves and no exception is thrown.
+
+Previously, attempting to remove failed image uploads with the `images_upload_handler` API failed if Real-time Collaboration was loaded.
+
+With this update, `images_upload_handler` removes failed image uploads and presents the correct alert message.
+
+Keyboard events can now be prevented before the Real-time Collaboration plugin intercepts them.
+
+Previously, selecting linked images when the `link_context_toolbar` option was enabled, caused the Real-time Collaboration editor to enter an endless loop. This loop caused the selected images to flicker.
+
+With this update, the Real-time Collaboration editor no longer enters an endless loop when linked images are selected. Consequenlty, the selected images no longer flicker.
+
+
+=== Spell Checker Pro
+
+The free `spellchecker` plugin was removed as no longer supported.
```

Comment:
I'm not sure we really need to note that in the premium docs 🤔 it doesn't impact spell checker pro at all.

Reply:
It’s mentioned primarily for technical marketing purposes.

The sub-text is _you were using the free spellchecker? Sorry, it’s now gone. This is the alternative._

Reply:
I can appreciate that in the core docs, but I'm not sure a free user will read the premium changes doc.

Reply:
I'd agree with Andy here, this makes little sense to be in a section about premium plugins.

Reply:
The consensus is in, and the above Sentence has been excised. (As well as the following sentence, which didn’t make sense once the first sentence was excised.)